files/nodebot.rb

```diff
@@ -4,24 +4,17 @@
 require "#{File.dirname(__FILE__)}/base"
 
 class Nodebot < BaseHandler
-  def pages_irc_channel
-    team_data('pages_irc_channel') || "##{team_name}-pages"
+  def pager_channel_keys
+    %w[ pages_irc_channel ]
```

Comment:
does this actually work? I don't see a `pages_irc_channel` method in your copy now

Reply:
sorry, i wrote this pre-coffee. i see the indirection now :)